tests/ipex/test_inference.py

```diff
@@ -54,27 +55,48 @@ class IPEXIntegrationTest(unittest.TestCase):
     )
 
     TEXT_GENERATION_SUPPORTED_ARCHITECTURES = (
-        "bloom",
```

Comment:
Why are these architectures now unsupported ?

Reply:
currently, bloom ang gpt_neox has some issue in jit, because we use tuple as jit input. I have a fix to support dict input in jit.trace, which will be released in torch 2.1. I add check in ci for jit to avoid regression like self.assertTrue(isinstance(ipex_text_generator.model._optimized, torch.jit.RecursiveScriptModule)), this two architextures will lead to fail in this assertion. will add some architecture back in 2.1

Reply:
previously this case could pass because it's silently switch to eager models when jit.trace fails

Reply:
I see, thanks for the fix @sywangyi !